blackbox/docs/config/secure.rst

```diff
@@ -0,0 +1,73 @@
+.. highlight:: sh
+
+.. _conf-secure-settings:
+
+===============
+Secure Settings
+===============
+
+Some settings are sensitive as they might store passwords, access keys or 
+other similar information. This type of data needs more protection than just 
+filesystem permissions and even more, should not be available in plain text.
+
+For these instances, we provide a keystore and the ``crate-keystore`` tool to 
+manage the settings in the keystore.
+
+.. NOTE::
+
+    Changes to the keystore take affect only after restarting CrateDB
+
+.. NOTE::
+
+    The CrateDB keystore currently provides obfuscation, but in the future 
+    password protection will be added.
+
+.. NOTE::
+
+    Not all settings are read from the keystore. Consult the documentation for 
+    each setting to see if it is secure setting.
+
```

Comment:
the note about the ownership is missing here, either change permissions later on so that the user crate runs with can read it, or create the keystore with that user